frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BrowseNavSection.tsx

```diff
@@ -83,6 +92,30 @@ export const BrowseNavSection = ({
           {t`Databases`}
         </PaddedSidebarLink>
       )}
+
+      <DelayedLoadingAndErrorWrapper
+        loading={areMetricsLoading}
+        error={metricsError}
+        loader={
+          <Flex py="sm" px="md" h="32.67px" gap="sm" align="center">
+            <Skeleton radius="md" h="md" w="md" />
+            <Skeleton radius="xs" w="4rem" h="1.2rem" />
+          </Flex>
```

Comment:
Maybe it's already time to extract this as a component. There's lots of hardcoded values here.